website2/src/components/terminal/styles.module.css

```
@@ -14,11 +14,20 @@
 .content {
   border: 1px solid var(--color-black);
   padding: 24px;
+  overflow: scroll
```

I think this should be auto, right?

```suggestion
  overflow: auto;
```